src/collectors.m4

```diff
@@ -57,14 +57,14 @@ m4_define([_pos_suffix], [[_pos]])
 m4_define([_arg_prefix], [[_arg_]])
 m4_define([_args_prefix], [[_args_]])
 m4_define([_varname], [m4_do(
-	[m4_quote(_arg_prefix[]_translit_var([$1]))],
+	[m4_quote(_arg_prefix[]_translate_var([$1]))],
```

Comment:
Kudos for having the courage to venture into the m4 code.

Reply:
Yup, it was interesting since I hadn't really played with m4 before.  I'm sure that was pretty apparent in my code that you fixed up.  Additionally, as I said earlier, I went back to translit, I think transliteration is a better fit after reading up some.